clang/lib/Driver/Driver.cpp

```diff
@@ -3195,9 +3195,12 @@ class OffloadingActionBuilder final {
     /// Flag to signal if the user requested device-only compilation.
     bool CompileDeviceOnly = false;
 
-    /// Flag to signal if the user requested the device object to be wrapped
+    /// Flag to signal if the user requested the device object to be wrapped.
     bool WrapDeviceOnlyBinary = false;
 
+    /// Flag to signal if the user requested do not perform device code split.
```

Comment:
According to the compiler options user can't request "do not perform device code split".

Reply:
Right, will fix.

Reply:
Yes, fixed.